packages/docs/src/utilities/utils.ts

```diff
@@ -1,25 +1,39 @@
 // File for documentation helper functions
 
+function getUIElementNameFromPath (path: string): string {
+  /**
+  * work's example:
+  * 'src\page-configs\ui-elements\component-name' --> 'component-name'
+  * why:
+  * it is only a guess that the path written through the relative operator "../"
+  * works differently in unix and windows and for fully identical operation
+  * in webpack we have to pass a more specific path as specified in the documentation
+  * https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import
+   * TODO: To remove this hack, we need to pass the name of the folder from which the webpack should import in each case instead of path
+  **/
+  return path.match(/[\\/]([\w-_]+)$/)?.[1] || path || ''
```

Comment:
Just wondering, if we can maybe do nothing here, and webpack will still import everything just in case (as it likely won't be able to determine what's getting loaded).

But up to you if you want to spend anymore time on that (curiosity), as the issue is low prio (almost no one uses windows).